test/unit/master/test_atomizer.py

```diff
@@ -1,37 +1,37 @@
-from unittest.mock import MagicMock
+from unittest.mock import Mock
 from app.master.atomizer import Atomizer, AtomizerError
 
-from app.project_type.project_type import ProjectType
 from app.util.process_utils import get_environment_variable_setter_command
 from test.framework.base_unit_test_case import BaseUnitTestCase
 
 
 _FAKE_ATOMIZER_COMMAND = 'find . -name test_*.py'
-_FAKE_ATOMIZER_COMMAND_OUTPUT = 'test_a.py\ntest_b.py\ntest_c.py\n'
+_FAKE_ATOMIZER_COMMAND_OUTPUT = '/tmp/test/directory/test_a.py\n/tmp/test/directory/test_b.py\n/tmp/test/directory/test_c.py\n'
 _SUCCESSFUL_EXIT_CODE = 0
 _FAILING_EXIT_CODE = 1
 
 
 class TestAtomizer(BaseUnitTestCase):
     def test_atomizer_returns_expected_atom_list(self):
-        mock_project = MagicMock(spec_set=ProjectType)
+        mock_project = Mock()
```

Comment:
I think you could keep the spec on `ProjectType` by doing `Mock(spec=ProjectType)`. Using spec instead of spec_set relaxes the restriction on setting attributes but still verifies that any mocked methods must be valid ProjectType methods.